samples/node/safety_settings.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  GoogleGenerativeAI,
+  HarmBlockThreshold,
+  HarmCategory,
+} from "@google/generative-ai";
+
+async function safetySettings() {
+  // [START safety_settings]
+  const genAI = new GoogleGenerativeAI(process.env.API_KEY);
+  const model = genAI.getGenerativeModel({
+    model: "gemini-1.5-flash",
+    safetySettings: [
+      {
+        category: HarmCategory.HARM_CATEGORY_HARASSMENT,
+        threshold: HarmBlockThreshold.BLOCK_LOW_AND_ABOVE,
+      },
+    ],
+  });
+
+  const unsafePrompt =
+    "I support Martians Soccer Club and I think " +
+    "Jupiterians Football Club sucks! Write a ironic phrase about them.";
+
+  const result = await model.generateContent(unsafePrompt);
+
+  try {
+    result.response.text();
+  } catch (e) {
+    console.error(e);
+    console.log(result.response.candidates[0].safetyRatings);
+  }
+  // [END safety_settings]
```

Comment:
This throws an error around half the time for me.

Reply:
I made a modification that seems to trigger it every time I've tried, let me know what you think.